docs/intro/acknowledgements.md

```diff
@@ -109,6 +109,8 @@ The sources and terms of reuse for these images are summarized in the table belo
   - CC-BY 4.0
 ::::
 
+***TODO*** Add TMA image to the table
```

Comment:
Add a line explaining that to access the images one has to follow the links, not download the miniature? There was some trouble with this at the Crick

Reply:
Noted for post I2K